app/views/account/admin/user/users.html.haml

```diff
@@ -0,0 +1,21 @@
+-provide(:page, 'users')
+-provide(:subpage, 'users')
+%div(class="row" )
+  %div(class="col-md-8")
+    =render "account/admin/user/_part/sub_menu"
+    %table(class="table")
+      %thead(class="table-secondary")
+        %tr(class="text-center" )
+          %th{ scope: "col", class: "w-50"} Name
+          %th(scope="col") Status
+          %th(scope="col") Actions
+      %tbody
+        - @users.each do |user|
+          %tr(scope="row")
+            %td= user.first_name + " " + user.last_name
```

Comment:
this should be a model method called `full_name` as it will be used very often around application